app/src/containers/Signup/Signup.js

```diff
@@ -1,9 +1,142 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { FormGroup, ControlLabel, FormControl, Button, ButtonToolbar, Form, HelpBlock } from 'react-bootstrap';
+import { browserHistory } from 'react-router';
+import style from './Signup.scss';
 
-export default class Signup extends React.Component {
-  render () {
+let request = require('superagent');
+let user = {firstName: 'Ivan', lastName: 'Popov', email: 'rrr@ukr.net'};
+
+const FieldGroup = ({className, label, inErrors, ...props}) => {
```

Comment:
Normally you would define a React component as a plain JavaScript class:
class Greeting extends React.Component {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}